Makefile

```diff
@@ -84,10 +86,34 @@ prepare:: linux-prepare-$(LINUX_DISTRO) linux-prepare-$(LINUX_DISTRO)-$(LINUX_DI
 		cat Documentation/binfmt_misc-warning-Linux.txt ; \
 	fi
 
+ifeq ($(NO_SUDO),false)
 linux-prepare-Ubuntu::
+	@echo
 	@echo Installing build depedencies for $(LINUX_DISTRO)
 	@echo Will use sudo, please provide your password as needed
+  @echo
```

Comment:
This line looks weird. Is the indent correct? Looks like a spaces vs. tabs issue.